tools/Utils.psm1

```diff
@@ -0,0 +1,181 @@
+﻿# There's a bug in Powershell where it doesn't properly handle bad paths in the LIB environment variable
+# when loading C# reference assemblies.  In order to work around that bug, we'll manually test the paths in LIB
+# and remove ones that don't actually exist.
+if ($env:LIB -ne $null -and $env:LIB.Length -gt 0)
+{
+    [System.Collections.Generic.List[string]]$libPaths = @()
+
+    foreach ($libPath in $env:LIB.Split(';', [System.StringSplitOptions]::RemoveEmptyEntries))
+    {
+        if (Test-Path $libPath)
+        {
+            $libPaths.Add($libPath)
+        }
+    }
+
+    $env:LIB = $libPaths -join ";"
+}
+
+Add-Type -Language CSharp @"
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.InteropServices.WindowsRuntime;
+
+namespace WinmdHelper
```

Comment:
Better name?

I *loathe* 'helper'. Right up there with 'help' and 'util'.

Suggestion: namespace Microsoft.WinMD

Reply:
Jevan?  Coordinate?  (I'm amenable).

Reply:
This is some internal script code, does it really matter? I'd rather not change it so we can more easily identify the similar files between reunion and MUX and then factor them out together.

Reply:
OK, won't fix.